osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -2,59 +2,67 @@
 using osu.Framework.Graphics.UserInterface;
 using osu.Game.Beatmaps.Timing;
+using osu.Game.Configuration;
 using osu.Game.Rulesets.Scoring;
 using osu.Game.Screens.Play.Break;
 
 namespace osu.Game.Screens.Play
 {
-    public class BreakOverlay : Container
-    {
-        private const double fade_duration = BreakPeriod.MIN_BREAK_DURATION / 2;
-        private const float remaining_time_container_max_size = 0.3f;
-        private const int vertical_margin = 25;
+  public class BreakOverlay : Container
+  {
+    public BindableFloat BackgroundOpacityOffset;
 
-        private List<BreakPeriod> breaks;
+    private const double fade_duration = BreakPeriod.MIN_BREAK_DURATION / 2;
+    private const float remaining_time_container_max_size = 0.3f;
+    private const int vertical_margin = 25;
 
-        private readonly Container fadeContainer;
+    private List<BreakPeriod> breaks;
 
-        public List<BreakPeriod> Breaks
-        {
-            get => breaks;
-            set
-            {
-                breaks = value;
-                initializeBreaks();
-            }
-        }
+    private Bindable<bool> lightenDuringBreaks;
+    private readonly BindableBool breakLighten = new BindableBool();
 
-        public override bool RemoveCompletedTransforms => false;
+    private readonly Container fadeContainer;
 
-        private readonly Container remainingTimeAdjustmentBox;
-        private readonly Container remainingTimeBox;
-        private readonly RemainingTimeCounter remainingTimeCounter;
-        private readonly BreakInfo info;
-        private readonly BreakArrows breakArrows;
+    public List<BreakPeriod> Breaks
+    {
+      get => breaks;
+      set
+      {
+        breaks = value;
+        initializeBreaks();
+      }
+    }
 
-        public BreakOverlay(bool letterboxing, ScoreProcessor scoreProcessor)
-            : this(letterboxing)
-        {
-            bindProcessor(scoreProcessor);
-        }
+    public override bool RemoveCompletedTransforms => false;
 
-        public BreakOverlay(bool letterboxing)
-        {
-            RelativeSizeAxes = Axes.Both;
-            Child = fadeContainer = new Container
-            {
-                Alpha = 0,
-                RelativeSizeAxes = Axes.Both,
-                Children = new Drawable[]
-                {
+    private readonly Container remainingTimeAdjustmentBox;
+    private readonly Container remainingTimeBox;
+    private readonly RemainingTimeCounter remainingTimeCounter;
+    private readonly BreakInfo info;
+    private readonly BreakArrows breakArrows;
+
+    public BreakOverlay(bool letterboxing, ScoreProcessor scoreProcessor)
+        : this(letterboxing)
+    {
+      bindProcessor(scoreProcessor);
+    }
+
+    public BreakOverlay(bool letterboxing)
+    {
+      RelativeSizeAxes = Axes.Both;
+      Child = fadeContainer = new Container
+      {
+        Alpha = 0,
+        RelativeSizeAxes = Axes.Both,
+        Children = new Drawable[]
+          {
```

Comment:
Nitpick: Indent is broken on the following lines (it's too far).

Reply:
I don't know how that happened but I'll fix it